subprojects/signing/src/main/java/org/gradle/plugins/signing/SigningExtension.java

```diff
@@ -305,7 +308,47 @@ public void execute(Sign task) {
         return result;
     }
 
-    private Sign createSignTaskFor(final CharSequence name, Action<Sign> taskConfiguration) {
+    /**
+     * Creates signing tasks that sign {@linkplain PublicationInternal#getPublishableArtifacts()} all publishable artifacts} of the given publications.
```

Comment:
❌ There should be no mention of internal APIs in the documentation.

Reply:
Fixed in 71838111765.